src/setup/serverless-config.go

```diff
@@ -141,6 +186,16 @@ func RemoveService(path string) string {
 	return slsRemoveMessage
 }
 
+// RemoveServiceAzure removes the Azure service defined in serverless.yml
+func RemoveServiceAzure(path string) string {
+	slsRemoveCmd := exec.Command("sls", "remove", "--force")
+	slsRemoveCmd.Dir = path
+	slsRemoveMessage := util.RunCommandAndLog(slsRemoveCmd)
+	// cleanup
+	util.RunCommandAndLog(exec.Command("rm", fmt.Sprintf("%sserverless.yml", path)))
+	return slsRemoveMessage
+}
+
```

Comment:
Put the switch case logic inside `main.go` into a `RemoveService(provider string, path string)` that calls these two functions, and call `RemoveService` in `main.go`.